Gemfile

```diff
@@ -79,7 +79,7 @@ gem 'pg'
 gem 'devise'
 gem 'devise-async'
 
-gem 'omniauth', '~> 1.3.2'
+gem 'omniauth', '~> 1.6.0'
```

Comment:
Why do you need this? Have you tested the oauth 2 authentication against your changes?

Reply:
This is does automaticly i have not tuch to the Gemfile